tornjak-frontend/src/components/entry-create-json.js

```diff
@@ -0,0 +1,825 @@
+                                                    }
+                                                    {this.state.newEntriesIds.length === 1 &&
+                                                        <div>
+                                                            {this.state.newEntriesIds.length} Entry Loaded from File
+                                                        </div>
+                                                    }
+                                                    {this.state.newEntriesIds.length > 1 &&
+                                                        <div>
+                                                            {this.state.newEntriesIds.length} Entries Loaded from File
+                                                        </div>
+                                                    }
+                                                </div>
+                                            </span>}
+                                        timeout={60000}
+                                        title="Entries Upload Notification"
+                                    />
+                                </div>
+                            }
+                        </div>
+                        <div className="view_entries_yaml_button">
+                            <ModalWrapper
+                                passiveModal={true}
+                                size='lg'
+                                triggerButtonKind="ghost"
+                                buttonTriggerText="View Uploaded Entries"
+                                modalHeading="Entries JSON"
+                                modalLabel="View Uploaded Entries"
+                            >
+                                <pre className="yaml_view_modal_json">{JSON.stringify(this.state.uploadedEntries, null, ' ')}</pre>
+                            </ModalWrapper>
+                        </div>
+                        <div className="edit_entries_button">
+                            <ModalWrapper
+                                size='lg'
+                                triggerButtonKind="ghost"
+                                buttonTriggerText="Edit Uploaded Entries"
+                                handleSubmit={this.applyEditToEntry}
+                                //shouldCloseAfterSubmit={true}
+                                modalHeading="Entries Edit"
+                                modalLabel="Edit Uploaded Entries"
+                                primaryButtonText="Apply"
+                                secondaryButtonText="Exit"
+                            >
+                                <div className='edit-entry-container'>
+                                    <div className="entries-list-container">
+                                        <fieldset>
+                                            <legend className="modal_Entry_list_title">1. SELECT ENTRY</legend>
```

Comment:
Can we do in larger text:

"STEP 1"
  "       Select Entry" 

Then on the right:
"STEP 2" 
   "       Edit Entry"

They both should be aligned, on the same horizontal level, Step 1 and 2

Can we add: 
"STEP 3"
"        APPLY"

We can create a warning/pop-up, if user selects another entry after making edits, without pressing APPLY 
"All changes will be lost! Press APPLY"

Reply:
Also, can we enhance the selected Entry? Maybe blue background and white font? To make it stand out better.

Reply:
done! The default carbon IBM link color is blue. So we have background color light grey and selected entry color blue.